01-arch/07-acl.md

```diff
@@ -103,6 +103,34 @@ and can be presented in different intermediate formats, like JSON, for the users
 }
 ```
 
+Note that some filters with `$Object` prefix are not suitable for making denying rules on certain operations. There may be an undefined behaviour on some combinations of NeoFS verbs and object attributes when eACL is set.
+
+|$Object:       |   GET   |  HEAD   |   PUT   | DELETE  | SEARCH  |  RANGE  |RANGEHASH|
+|---------------|---------|---------|---------|---------|---------|---------|---------|
+|version        |    v    |    v    |    v    |    -    |    -    |    -    |    -    |
```

Comment:
Need to make a clarification on what does `v` and `-` mean here.

Reply:
We can decide on better marks.
What about using `OK` for `v` and `UB` (as in Undefined Behavior) for `-`?